### DISCUSSION TITLE
Download with multiple audio

### DISCUSSION THREAD

**Discussion by aragorn74** [aclap-dev/video-downloadhelper#1441] [Help and Q&A]
### Your question

I finally found the program to download the videos I am interested in.
The problem is that the video has several audio streams in different languages; even if I select Italian as the main audio language, when downloading the program downloads the video with the first language in the list.

### Addon & CoApp version and details

![image](https://github.com/user-attachments/assets/16e6c937-65f8-4ed4-a722-3062e1d8103e)

**Comment by Mattmjs** [comment]
Where is this happening ? do you have an example.

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

You might want to look at #956.

**Comment by computer-guru** [comment]
Here is an example of a youtube video with 2 audio tracks. The first track is english and the second track is spanish. The video only downloads in spanish for me. There needs to be UI element for selecting the desired audio track.

https://www.youtube.com/watch?v=f-ldPgEfAHI

**Reply by Mattmjs** [reply]
Use a youtube download site or yt-dlp to get it with english.

**Reply by computer-guru** [reply]
Please expand on your answer. Are you saying the VDH cannot do this task and another tool should be used?

**Reply by Mattmjs** [reply]
The fact is as another user and not a developer of the software, it has no solution yet when a video has multiple audio tracks.
So it's not able to do what you want. You will just have to find another way.
